pyproject.toml

```
@@ -45,7 +45,7 @@ license = "MIT"
 name = "hume"
 packages = [{ include = "hume", from = "src" }]
 readme = "README.md"
-version = "0.7.0-rc2"
+version = "0.6.1"
```

```suggestion
version = "0.7.0-rc3"
```
Would specifying `0.6.1` imply that the default version installed via `pip install hume` now becomes the Fern-generated SDK with these mic utilities? If so, then I highly suggest we instead release a pre-release candidate at this point (e.g., `0.7.0-rc3`), and then change it to `0.6.1` or other version number after all supplementary materials are complete and ready to be published simultaneously. Without that, users would find it jarring to see the breaking changes.

---

Great catch